web/src/ApiButton.tsx

```diff
@@ -1,15 +1,38 @@
-import { Icon, SvgIcon } from "@material-ui/core"
+import { ButtonGroup, Icon, SvgIcon, TextField } from "@material-ui/core"
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown"
 import moment from "moment"
-import React, { useState } from "react"
+import React, { useRef, useState } from "react"
 import { convertFromNode, convertFromString } from "react-from-dom"
+import styled from "styled-components"
+import FloatDialog from "./FloatDialog"
 import { InstrumentedButton } from "./instrumentedComponents"
+import { Color, FontSize, SizeUnit } from "./style-helpers"
 
 type UIButton = Proto.v1alpha1UIButton
+type UIInputSpec = Proto.v1alpha1UIInputSpec
+type UIInputStatus = Proto.v1alpha1UIInputStatus
+
+const ApiButtonFormRoot = styled.span`
```

Comment:
you really shouldn't put block elements (like div) inside inline elements (like span) - browsers will behave somewhat erratically when you do this. Here's a good doc:
https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements